RFC.md

```diff
@@ -1,9 +1,18 @@
 # RFC: Amplify JS TypeScript Improvements
+Amplify JS is looking to improve our TypeScript support across the library to better serve our customers and provide a more intuitive & idiomatic developer experience. To this end we are requesting community feedback on a variety of changes and improvements that will be available to customers in the near future.
 
-# General TypeScript Improvements
+This RFC is broken down into sections covering:
+- Library-wide TypeScript improvements
+- Changes to some of our core utilities such as `Hub` and Amplify configuration
+- Specific improvements to the `Auth`, `Storage`, and `API` categories and associated APIs 
```

Comment:
I would mention that we still want feedback for the other categories types as well